examples/simple_system/README.md

```diff
@@ -6,6 +6,7 @@ run stand-alone binaries. It contains:
 * An Ibex Core
 * A single memory for instructions and data
 * A basic peripheral to write ASCII output to a file and halt simulation from
+* A basic timer peripheral capable of generating interrupts based on the RISC-V Machine Timer Register (see RISC-V Privileged Specification, version 1.11, Section 3.1.10)
   software
```

Comment:
I think this stray software belongs with the bullet point on line 8?

Reply:
Fixed